src/ucp/wireup/select.c

```diff
@@ -1566,6 +1595,79 @@ ucp_wireup_select_params_init(ucp_wireup_select_params_t *select_params,
     select_params->show_error    = show_error;
 }
 
+static double
+ucp_wireup_keepalive_score_func(ucp_worker_h worker, ucp_rsc_index_t rsc_index,
+                                const uct_md_attr_t *md_attr,
+                                const uct_iface_attr_t *iface_attr,
+                                const ucp_address_iface_attr_t *remote_iface_attr)
+{
+    ucp_context_h context                  = worker->context;
+    ucp_worker_iface_t *wiface             =
+            ucp_worker_iface(worker, rsc_index);
+    uct_perf_attr_t perf_attr              = {
```

Comment:
the struct is mutable, better not init inline

Reply:
done